package/network/utils/wireguard-tools/Makefile

```
@@ -35,8 +35,8 @@ define Package/wireguard-tools
   MAINTAINER:=Jason A. Donenfeld <Jason@zx2c4.com>
   TITLE:=WireGuard userspace control program (wg)
   DEPENDS:= \
-	  +@BUSYBOX_CONFIG_IP \
-	  +@BUSYBOX_CONFIG_FEATURE_IP_LINK \
+	  +!BUSYBOX_CONFIG_IP:ip-full \
+	  +!BUSYBOX_CONFIG_FEATURE_IP_LINK:ip-full \
```

Is it possible to select `ip` instead? 
`ip-tiny` and `ip-full` are providing `ip`. wireguard hopefully also works with `ip-tiny`.
```suggestion
	  +!BUSYBOX_CONFIG_FEATURE_IP_LINK:ip \
```

If the system works like I expect it will select `ip-tiny` if none of the 3 options is available, but work also with `ip-full` and the busybox version. 

Please test that it selects the expected packages correctly.

---

It seems like this indeed works as you suggest - if wireguard-tools is brought in and no `ip` providing package at all is selected, then `ip-tiny` is force-included. If I select `ip-full` or add the `ip` applet to Busybox, `ip-tiny` is no longer required and indeed replaced by the other packages. Seems like the best possible solution!